### PR TITLE
utils: fix help example in m0spiel

### PR DESCRIPTION
**Pull request by andriytk**
The function should be service_init(), not spiel_service_init() in the example.

### CODE REVIEW

File: utils/spiel/m0spiel [hunk 1]
```diff
@@ -700,7 +700,7 @@ The following example illustrates usage of the tool:
 
 Execute commands from command line:
 
-\techo 'print(spiel_service_init(Fid(1,1)))' | sudo ./m0spiel''')
+\techo 'print(service_init(Fid(1,1)))' | sudo ./m0spiel''')
```
Comment (huanghua78) [inline utils/spiel/m0spiel:703]: Please add some empty lines after this line.
This line and above lines are help messages.
And the next line is start argument parsing.
It's better to have some empty lines to help reading.

Comment (andriytk) [reply utils/spiel/m0spiel]: Done, thanks.

**Review by huanghua78 — APPROVED** [review]
Looks good.

**Review by ivan-alekhin — APPROVED** [review]

**Review by mehjoshi — APPROVED** [review]
Looks good to me.

**Pull request merged**